rotimi-best/classroomio
# fix: exercise points must accept only numbers

## What does this PR do?



This PR addresses a points validation error where incorrect or non-numeric values were accepted in the points input field. It resolves issue #380, which caused issues during form submission.
Fixes # 380



https://github.com/rotimi-best/classroomio/assets/109029101/cfece441-13c0-48c9-acf6-f0667a889c10



## Type of change




## How should this be tested?



- Test A 
Navigate to the form where points are entered (e.g., creating a new exercise).
Enter non-numeric values (e.g., text, symbols) into the points input field.
Verify that an error message appears stating "Points should be a number only."
Enter valid numeric values and confirm that the error message disappears.

- Test B
Fill out the form to create a new exercise, including entering valid numeric points.
Submit the form.
Verify that the form submits successfully without any validation errors related to the points field.
Check that the created exercise reflects the correct points value entered.

## Checklist



### Required


### Appreciated
